src/api/getSingleMessage.js

```diff
@@ -0,0 +1,12 @@
+import { apiGet } from './apiFetch';
+
+export default async (
+  auth,
+  messageId,
+) =>
+  apiGet(
+    auth,
+    `messages/${messageId}`,
+    {},
+    res => res,
```

Comment:
Yes, we are interested only for this but I think its worth to keep the version like it is for future purposes. Its API call only so I think it could provide more info - or maybe its somehow wrong from the performance point of view?